airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/constants/GlobalDataSizeConstants.java

```diff
@@ -0,0 +1,15 @@
+package io.airbyte.integrations.destination.jdbc.constants;
+
+import io.aesy.datasize.ByteUnit.IEC;
+import io.aesy.datasize.DataSize;
+
+public interface GlobalDataSizeConstants {
+  /** 256 MB to BYTES as comparison will be done in BYTES */
+  int DEFAULT_MAX_BATCH_SIZE_BYTES = DataSize.of(256L, IEC.MEBIBYTE).toUnit(IEC.BYTE).getValue().intValue();
+  /** This constant determines the max size of uploaded chunk of file(e.g. 15 GB / 256 megabytes ≈ 58 MB per chunk of file)
+  see StagingFilenameGenerator.java:28
+  */
+  long MAX_BYTE_PARTS_PER_FILE_DEFAULT = DataSize.of(15L, IEC.GIBIBYTE).toUnit(IEC.BYTE).getValue().longValue();
+
+  long MAX_BYTE_PARTS_PER_FILE_SNOWFLAKE = DataSize.of(700L, IEC.MEBIBYTE).toUnit(IEC.BYTE).getValue().longValue();
```

Comment:
nitpick: this constant should be in destination-snowflake. Maybe just put it directly into `SnowflakeGcsStreamCopierFactory`?

Reply:
also, why are we targeting 700MiB per part?

Reply:
also also, does this address https://github.com/airbytehq/airbyte/issues/6517 ?

Reply:
1. I have moved it to SnowflakeGcsStreamCopierFactory. However, in such case we need to include additional library to destination-snowflake module. 
2. I believe I don't used MAX_BYTE_PARTS_PER_FILE_SNOWFLAKE for proper calculation, I have changed the formula now, please take a look.
3. Potentially, this PR could be a fix. To be honest, need to measure performance, at least from local machine to see if results are positive.

Reply:
1. sounds good, I think that's fine
2. added a comment on the StagingFileGenerator
3. makes sense. would you prefer to keep that separate from this PR?

Reply:
3. Yep, I believe we can create the PR in scope of 6517.